src/components/chip-set/chip-set.tsx

```diff
@@ -513,22 +519,27 @@ export class ChipSet {
         }
 
         return (
-            <div
-                class={{
-                    'mdc-chip': true,
-                    'mdc-chip--selected': this.inputChipIndexSelected === index,
-                    disabled: this.disabled,
-                }}
-                role="row"
-                id={`${chip.id}`}
-                onClick={this.catchInputChipClicks}
-                {...attributes}
-            >
-                {chip.icon ? this.renderIcon(chip) : null}
-                {this.renderLabel(chip)}
-                {chip.removable && !this.readonly && !this.disabled
-                    ? this.renderTrailingIcon()
-                    : null}
+            <div class="chip-delimiter-wrapper">
+                {index !== 0 && this.delimiter !== null ? (
+                    <div class="delimiter">{this.delimiter}</div>
+                ) : null}
```

Comment:
I think this will be a little easier to read if we refactor this into it's own render method :slightly_smiling_face: